content/docs/reference/pkg/python/pulumi/_index.md

```diff
@@ -905,7 +905,7 @@ are resolved asynchronously, but they also contain information about the depende
 program.</p>
 <p>Pulumi does not offer direct access to the values contained within Outputs. Instead, you must use the <cite>apply</cite> function
 on the Output class in order to observe the value of an output. See
-<a class="reference external" href="/tour/programs-properties.html">the documentation</a> for more details on this part of the Pulumi programming model.</p>
+<a class="reference external" href="/docs/reference/programming-model/#outputs">the documentation</a> for more details on this part of the Pulumi programming model.</p>
```

Comment:
From the header, it doesn't look like this file is auto-generated, but it definitely doesn't look like it was written by a human.

Do we need to make this change somewhere else in source code for if this file ever gets regenerated (if that is the case)?

Reply:
The template (that this is generated from) was also changed in this PR: https://github.com/pulumi/docs/pull/1462/files#diff-d5bf7c984b191b89c451585290f7d7cc